packages/autorest.gotest/test/integrationtest/output/compute/armcompute/virtualmachines_client_example_test.go

```diff
@@ -40,6 +40,241 @@ func ExampleVirtualMachinesClient_NewListByLocationPager() {
+		// 					"computerName": "Test",
+		// 					"secrets": [],
+		// 					"windowsConfiguration": {
+		// 						"enableAutomaticUpdates": true,
+		// 						"provisionVMAgent": true
+		// 					}
+		// 				},
+		// 				"provisioningState": "Succeeded",
+		// 				"storageProfile": {
+		// 					"dataDisks": [],
+		// 					"imageReference": {
+		// 						"offer": "WindowsServer",
+		// 						"publisher": "MicrosoftWindowsServer",
+		// 						"sku": "2012-R2-Datacenter",
+		// 						"version": "4.127.20170406"
+		// 					},
+		// 					"osDisk": {
+		// 						"name": "test",
+		// 						"caching": "None",
+		// 						"createOption": "FromImage",
+		// 						"diskSizeGB": 127,
+		// 						"osType": "Windows",
+		// 						"vhd": {
+		// 							"uri": "https://{storageAccountName}.blob.core.windows.net/{containerName}/{vhdName}.vhd"
+		// 						}
+		// 					}
+		// 				},
+		// 				"vmId": "{vmId}"
+		// 			},
+		// 			"tags": {
+		// 				"RG": "rg",
+		// 				"testTag": "1"
+		// 			}
+		// 		},
+		// 		{
+		// 			"name": "{virtualMachineName}",
+		// 			"type": "Microsoft.Compute/virtualMachines",
+		// 			"id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{virtualMachineName}",
+		// 			"location": "eastus",
+		// 			"properties": {
+		// 				"availabilitySet": {
+		// 					"id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/availabilitySets/{availabilitySetName}"
+		// 				},
+		// 				"hardwareProfile": {
+		// 					"vmSize": "Standard_A0"
+		// 				},
+		// 				"networkProfile": {
+		// 					"networkInterfaces": [
+		// 						{
+		// 							"id": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkInterfaces/{networkInterfaceName}"
+		// 						}
+		// 					]
+		// 				},
+		// 				"osProfile": {
+		// 					"adminUsername": "Foo12",
+		// 					"allowExtensionOperations": true,
+		// 					"computerName": "Test",
+		// 					"secrets": [],
+		// 					"windowsConfiguration": {
+		// 						"enableAutomaticUpdates": true,
+		// 						"provisionVMAgent": true
+		// 					}
+		// 				},
+		// 				"provisioningState": "Succeeded",
+		// 				"storageProfile": {
+		// 					"dataDisks": [],
+		// 					"imageReference": {
+		// 						"offer": "WindowsServer",
+		// 						"publisher": "MicrosoftWindowsServer",
+		// 						"sku": "2012-R2-Datacenter",
+		// 						"version": "4.127.20170406"
+		// 					},
+		// 					"osDisk": {
+		// 						"name": "test",
+		// 						"caching": "None",
+		// 						"createOption": "FromImage",
+		// 						"diskSizeGB": 127,
+		// 						"osType": "Windows",
+		// 						"vhd": {
+		// 							"uri": "https://{storageAccountName}.blob.core.windows.net/{containerName}/{vhdName}.vhd"
+		// 						}
+		// 					}
+		// 				},
+		// 				"vmId": "{vmId}"
+		// 			},
+		// 			"tags": {
+		// 				"RG": "rg",
+		// 				"testTag": "1"
+		// 			}
+		// 		}
+		// 	]
+		// }
```

Comment:
@xboxeer @ArcturusZhang It seems too long if we comment all the response struct and JSON. Any suggestion?

Reply:
is this the longest or just "middle size"?

Reply:
I'm not sure. Perhaps compute has the most complicated examples. You could review all the `**/*_example_test.go` files [here](https://github.com/Azure/autorest.go/tree/a51dd41438a735d97b7187b48c5d4daaf88f4692/packages/autorest.gotest/test/integrationtest/output) to see whether they are what you want.

Reply:
Others like signalR seems to be a reasonable length, let's roll this out first and see what is the feedback there

Reply:
OK. Any concern? @ArcturusZhang @lirenhe

Reply:
could you help to explain why we need the raw response? Is it from customer's ask?

Reply:
> could you help to explain why we need the raw response? Is it from customer's ask?

exactly, customer would like copy json output in their unit test.

Reply:
For this case, can we suggest them to use swagger example directly?

Reply:
> For this case, can we suggest them to use swagger example directly?

Each example has a comment point to the swagger example. But it seems not intuitional for the customers.

Reply:
> For this case, can we suggest them to use swagger example directly?

For customers who are not familiar with REST API, they would be uncomfortable to look for output in REST API Reference site
Another thing is that versioning, REST API reference site currently only support latest version API while SDK is fixed to one version API, we don't want bring extra information to customer about the versioning thing.